pcsx2/CDVD/CDVD.cpp

```diff
@@ -1048,9 +1048,9 @@ u8 cdvdRead(u8 key)
 			CDVD_LOG("cdvdRead07(Break) %x", 0);
 			return 0;
 
-		case 0x08:  // STATUS
-			CDVD_LOG("cdvdRead08(Status) %x", cdvd.Status);
-			return cdvd.Status;
```

Comment:
It does, seems the wrong way around tho xD You'd expect 0x0A to be the copy pasta of 0x08 lol